src/test/javascript/spec/component/exercises/shared/exercise-scores.component.spec.ts

```diff
@@ -1,24 +1,16 @@
-import { MockRouter } from '../../../helpers/mocks/mock-router';
 import { ArtemisTestModule } from '../../../test.module';
 import { MockHasAnyAuthorityDirective } from '../../../helpers/mocks/directive/mock-has-any-authority.directive';
 import { ArtemisTranslatePipe } from 'app/shared/pipes/artemis-translate.pipe';
 import { ArtemisDatePipe } from 'app/shared/pipes/artemis-date.pipe';
-import { MockTranslateService, TranslateTestingModule } from '../../../helpers/mocks/service/mock-translate.service';
-import { MockAlertService } from '../../../helpers/mocks/service/mock-alert.service';
 import { ExerciseScoresComponent } from 'app/exercises/shared/exercise-scores/exercise-scores.component';
 import { ResultService } from 'app/exercises/shared/result/result.service';
 import { ProgrammingSubmissionService } from 'app/exercises/programming/participate/programming-submission.service';
 import { ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testing';
 import { RouterTestingModule } from '@angular/router/testing';
-import { ActivatedRoute, Router } from '@angular/router';
-import { AlertService } from 'app/core/util/alert.service';
-import { MockComponent, MockDirective, MockPipe } from 'ng-mocks';
-import { TranslateService } from '@ngx-translate/core';
-import * as sinon from 'sinon';
-import sinonChai from 'sinon-chai';
-import * as chai from 'chai';
+import { ActivatedRoute } from '@angular/router';
+import { MockComponent, MockDirective, MockPipe, MockModule } from 'ng-mocks';
 import { of, Subscription } from 'rxjs';
-import { FormsModule } from '@angular/forms';
+import { NgModel, NgSelectOption } from '@angular/forms';
```

Comment:
```suggestion
import { NgModel } from '@angular/forms';
```
Just `NgModel` should be enough for this component, as there are no `select` drop-down menus in the HTML.